### PR TITLE
don't try to upgrade queue config table if does not exists

### PR DESCRIPTION
**Pull request by rohitsinha54**
If the queue config table does  not exists just skip the upgrade as its not  needed. 

JIRA: https://issues.cask.co/browse/CDAP-1814

### CODE REVIEW

File: cdap-master/src/main/java/co/cask/cdap/data/tools/QueueConfigUpgrader.java [hunk 1]
```diff
@@ -51,19 +53,25 @@
 public class QueueConfigUpgrader extends AbstractUpgrader {
   private static final Logger LOG = LoggerFactory.getLogger(QueueConfigUpgrader.class);
   private final HBaseTableUtil tableUtil;
+  private final Configuration conf;
 
   @Inject
   public QueueConfigUpgrader(LocationFactory locationFactory, HBaseTableUtil tableUtil) {
     super(locationFactory);
     this.tableUtil = tableUtil;
+    this.conf = HBaseConfiguration.create();
```
Comment (bdmogal) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/QueueConfigUpgrader.java]: Is it possible to inject ``HBaseConfiguration``?

**Comment by bdmogal** [comment]
Just one quick question, LGTM otherwise

**Comment by rohitsinha54** [comment]
Build : http://builds.cask.co/browse/CDAP-RBT149-1

**Comment by bdmogal** [comment]
LGTM, presuming this particular use case was tested.

**Comment by rohitsinha54** [comment]
New build :http://builds.cask.co/browse/CDAP-RBT149-3

**Comment by rohitsinha54** [comment]
Build passed here: http://builds.cask.co/browse/CDAP-RBT149-3

**Pull request merged**